.github/workflows/run_tests.yml

```diff
@@ -0,0 +1,41 @@
+name: Python tests
+
+on: [push, pull_request]
+
+jobs:
+  unit-tests:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install the package and dependencies
+      run: python setup.py install
+    - name: Lint with flake8
+      run: |
+        pip install flake8
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+    - name: Install testing dependencies
+      run: |
+        pip install -r requirements.txt
+        pip install coveralls coverage
+        pip install -U setuptools
+        pip install pytest --upgrade
+    - name: Test with pytest and run coverage
+      run: coverage run --source flask_combo_jsonapi -m pytest -v
+
+  trigger-coveralls:
```

Comment:
This is missing `runs-on`

I think it's just better to run this as next step in unit-tests job

Otherwise it'll run in other container and you'll have no previous coverage data or dependencies installed